app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
You are HAML phobic?  We actually prefer SLIM these days.
http://slim-lang.com/

Html ERB is definitely dying... and so is HAML.